### ISSUE TITLE
Folders containing a single document are not browsable

### ISSUE THREAD

**Issue by averyvh** [jellyfin/jellyfin#5958]
**Describe the bug**
In book libraries with directories containing a single pdf document and subdirectories, the directory and subdirectories are not browsable, because clicking on the directory just opens the pdf document inside it. This is still true when navigating to the directory from the "Folders" menu option.

**System (please complete the following information):**
 - OS: ArchLinux
 - Virtualization: none
 - Clients: Browser
 - Browser: Firefox 88
 - Jellyfin Version: 10.7.2
 - Reverse Proxy: none
 - Storage: local

**Comment by SjoerdV** [comment]
Yes, folder navigation needs an improvement IMHO: https://github.com/jellyfin/jellyfin/issues/9221#issuecomment-1513268044

**Comment by nonobio** [comment]
Hi,
It is similar to my issue I think: https://github.com/jellyfin/jellyfin/issues/9221#issuecomment-1636713205

**Comment by nonobio** [comment]
Hi,
Unless I'm mistaken, the problem is still present.

**Comment by CatSamaDD** [comment]
problem is still present in10.9.2

**Comment by nonobio** [comment]
still present in 10.9.11

**Comment by felix920506** [comment]
This lacks reproduction steps. Please provide concrete steps that will reliably reproduce the issue in the current release.

**Comment by SjoerdV** [comment]
> This lacks reproduction steps. Please provide concrete steps that will reliably reproduce the issue in the current release.

Reproduction: https://github.com/jellyfin/jellyfin/issues/9221#issuecomment-1513268044

**Comment by nonobio** [comment]
Still present

**Comment by felix920506** [comment]
Closing this one in favor of #9221 because these are essentially the same but that one contains significantly more useful information than this thread.
